editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -5,8 +5,10 @@ import Typography from "@mui/material/Typography";
 import Card from "@planx/components/shared/Preview/Card";
 import QuestionHeader from "@planx/components/shared/Preview/QuestionHeader";
 import { PublicProps } from "@planx/components/ui";
-import { GeoJSONObject } from "@turf/helpers";
-import { useStore } from "pages/FlowEditor/lib/store";
+import area from "@turf/area";
```

Comment:
Feel like I'm maybe just missing something here, but presumably we can't just get the area from the `areaChangeHandler` if there's a pre-loaded GeoJSON? Is this just throwing an event on change of the boundary?